frontend/app/src/components/home/MessageEntry.svelte

```diff
@@ -455,6 +460,11 @@
         </div>
     {:else if preview}
         <div class="preview">
+            {#if isSuperAdmin}
+                <Button secondary={true} small={true} on:click={freezeGroup}>
```

Comment:
You might also want to join the group to see more details and then should also be able to freeze from the group menu